### ISSUE TITLE
Aria-expanded on wrong element in facets (severity 5)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1614]
This will be a Blacklight issue.

The structure of the facets is currently something like this:

```
<div class="panel-heading" aria-expanded="false">
  <h3>
    <a>Facet</a>
  </h3>
</div>
<div id="facet-name" class="facet-content collapse" aria-expanded="false"> ... </div>
```

- _neither_ of the current `<div>`s (panel-heading and facet-content) should have `aria-expanded`
- `aria-expanded` should be on the element that **is** the toggle - in this case, the `<a>` tag
- the `aria-controls` attribute can be added to the toggle to link it to the content that will change - in this case, the facet-content `<div>`

```
<div class="panel-heading">
  <h3>
    <a aria-expanded="false" aria-controls="facet-name">Facet</a>
  </h3>
</div>
<div id="facet-name" class="facet-content collapse"> ... </div>
```

**Comment by jvine** [comment]
This has been resolved in BL 7, closing.
